ginga/qtw/QtHelp.py

```diff
@@ -170,19 +176,52 @@ def setSpacing(self, val):
 
 
 class FileSelection(object):
+    """Handle Load Image file dialog from File menu."""
+    def __init__(self, parent_w):
+        self.parent = parent_w
+        self.cb = None
+
+    def popup(self, title, callfn, initialdir=None, filename=None):
+        """Let user select and load file(s). This allows wildcards and
+        extensions, like in FBrowser.
+
+        Parameters
+        ----------
+        title : str
+            Title for the file dialog.
+
+        callfn : func
+            Function used to open the file(s).
+
+        initialdir : str or `None`
+            Directory for file dialog.
 
-    def __init__(self, parent_w, action=None):
-        # Create a new file selection widget
-        self.filew = QtGui.QFileDialog(parent_w, directory=os.curdir)
-        #self.filew.setFileMode(QtGui.QFileDialog.ExistingFile)
-        self.filew.setViewMode(QtGui.QFileDialog.Detail)
+        filename : str
+            Filter for file dialog.
 
-    def popup(self, title, callfn, initialdir=None,
-              filename=None):
+        """
         self.cb = callfn
-        if self.filew.exec_():
-            filename = list(map(str, list(self.filew.selectedFiles())))[0]
-            self.cb(filename)
+        filenames = QtGui.QFileDialog.getOpenFileNames(
+            self.parent, title, initialdir, filename)
```

Comment:
Ops, my bad! From what I see at https://www.reddit.com/r/learnpython/comments/2xhagb/pyqt5_trouble_with_openinggetting_the_name_of_the/ , I added a commit that theoretically fixed this. I cannot test this myself because Anaconda does not appear to support Qt5 yet (I cannot find it at http://docs.continuum.io/anaconda/pkg-docs).